tests/integration/data_sources_and_expectations/expectations/test_expect_column_values_to_be_in_type_list.py

```diff
@@ -76,31 +76,6 @@ def test_success_complete(batch_for_datasource: Batch) -> None:
     assert result_dict["observed_value"] in type_list
 
 
-@pytest.mark.xfail
-@parameterize_batch_for_data_sources(
-    data_source_configs=[DatabricksDatasourceTestConfig()],
-    data=DATA,
-)
-def test_success_complete_errors(batch_for_datasource: Batch) -> None:
```

Comment:
This test could be passing by adding `INT` to the array, but we already have a "success" test below this for Databricks. It's not clear how this is meant to test "errors". This is the only xfailed test for the type Expectations.